### ISSUE TITLE
install in python virtual environment?

### ISSUE THREAD

**Issue by oldfart101** [OttoWinter/esphomeyaml#54]
Home Assistant running in python virtual enviroment on Hassbian on PI3, Python 3.5.3
how to install??

**Comment by brandond** [comment]
Same as any other Python module?
```bash
source /path/to/venv/bin/activate
pip install esphomeyaml
```

Some folks have reported that the espressif toolchain (compiler, etc) tends to crash on arm, though.

**Comment by oldfart101** [comment]
pip install esphomeyaml
Collecting esphomeyaml
  Could not find a version that satisfies the requirement esphomeyaml (from versions: )
No matching distribution found for esphomeyaml

**Comment by oldfart101** [comment]
Yes, I had read the requirements for esphomeyaml, which is why I put Python 3.5.3 in the first post. So there is no way to install in my Virtual environment, since it is running Python3, can it be run outside the environment and connect to HA?

**Comment by brandond** [comment]
You'd need to create a separate virtualenv using Python 2.7. esphomeyaml usually interacts with HA via MQTT (over the network), so there's no reason they would need to be installed in the same virtualenv or even on the same host.
